spec/models/application_spec.rb

```diff
@@ -274,7 +241,7 @@
 
   describe "versioning", :versioning do
     context "when changing versioned fields" do
-      let(:application) { create(:application, lead_provider_approval_status: "pending", participant_outcome_state: nil) }
+      let(:application) { create(:application, lead_provider_approval_status: "pending", participant_outcome_state: nil, funded_place: false) }
```

Comment:
funded place is `nil` if the application is pending?

Reply:
removed, this isn't needed